### ISSUE TITLE
Cheatsheet request: jq

### ISSUE THREAD

**Issue by glitchim** [rstacruz/cheatsheets#64]
https://stedolan.github.io/jq/ would be a nice addition, thank you!

**Comment by rstacruz** [comment]
Hmm, I could've sworn I've already written one before... seems like I haven't. Let me see what I can do, PR's welcome :)

FWIW, the official https://stedolan.github.io/jq/tutorial/ is already almost cheatsheet-like

**Comment by Gerst20051** [comment]
I agree JQ has amazing documentation. Can't hurt to also have some here!
